themes/Clearlooks/lxqt-panel.qss

```diff
@@ -1,576 +1,815 @@
 /*
 "Clearlooks" LXQt theme
-By Azuma Hazuki (wanderinghazuki@gmail.com)
-Created 28 May 2020
+By Azuma Hazuki (wandering hazuki at gmail.com)
+Refactored on 23 Jan 2024
 */
-
-LXQtPanel #BackgroundWidget {
-    background-color: #edeceb;
-    border: 0px solid #1a1a1a;
+LXQtPanel, LXQtPanel #BackgroundWidget {
+    color: #1a1a1a;
+    background: #edeceb;
+}
+
+/* Panel Borders by position */
+LXQtPanel[position="Bottom"] #BackgroundWidget { border-top: 1px solid #ffffff; }
+LXQtPanel[position="Left"] #BackgroundWidget { border-right: 1px solid #ffffff; }
+LXQtPanel[position="Right"] #BackgroundWidget { border-left: 1px solid #ffffff; }
+
+/* Catch-all settings for panel plugins and their subwidgets 
+    not otherwise styled in this theme: 
+    #ColorPicker
+    #Custom and #CustomButton
+    #DirectoryMenu
+    #Dom
+    #LXQtMount
+    #NetworkMonitor
+    #Sensors
+    #ShowDesktop
+    #Spacer
+*/ 
+Plugin, Plugin > QToolButton, Plugin > QWidget > QToolButton,
+Plugin > QWidget > QToolButton > QWidget > QToolButton {
+    color: #1a1a1a;
+    background: #edeceb;
+    padding: 0px;
+    margin: 0px;
+    /* prevents unexpected borders/boxes on hover */
```

Comment:
That doesn't work. Showdesktop, colorpicker, main|fancymenu, quicklaunch items have now borders on hover, while some others don't (volume, mount, brightness).